doc/sphinx-guides/source/installation/config.rst

```diff
@@ -557,6 +557,11 @@ Reported Working S3-Compatible Storage
   **Can be used for quick testing, too:** just use the example values above. Uses the public (read: unsecure and
   possibly slow) https://play.minio.io:9000 service.
 
+'StorJ Integration'
+Per the general Dataverse Installation to use S3 Storage noted above, you’ll first set up the StorJ S3 store, by defining the id, type, and label. After Following the general installation set the following configurations to use a StorJ object store - dataverse.files.<id>.payload-signing=true This optional setting will enable payload signing for the S3 bucket. Set dataverse.files.<id>.chunked-encoding=false. This optional setting will disable chunked encoding.
```

Comment:
I think a link and perhaps some brief description, e.g. StorJ is a distributed object store that can be configured with a S3 gateway, would be useful.

Reply:
@dhirst1 it's probably easier to show than tell. I just create a pull request against this pull request with some tweaks. If you like it, please go ahead and merge it or steal what you want from it: https://github.com/dhirst1/dataverse/pull/1

Here's a preview of how it looks:

<img width="723" alt="Screen Shot 2022-01-06 at 10 57 20 AM" src="https://user-images.githubusercontent.com/21006/148412258-fb5de36a-8d4a-48c2-adc1-6d27f642507f.png">

If you don't have Sphinx installed to preview changes with `make html` I highly recommend it. We can help you get it installed if you have trouble. Thanks!